mdtraj/geometry/hbond.py

```diff
@@ -123,10 +127,13 @@ def wernet_nilsson(traj, exclude_water=True, periodic=True):
         raise ValueError('wernet_nilsson requires that traj contain topology '
                          'information')
 
-    def get_donors(e0, e1):
+    def get_donors(e0, e1, proposed_donor_indices=None):
         elems = set((e0, e1))
         bonditer = traj.topology.bonds
-        atoms = [(b[0], b[1]) for b in bonditer if set((b[0].element.symbol, b[1].element.symbol)) == elems]
+        if proposed_donor_indices==None:
```

Comment:
It's better to use `if x is None` when looking for None values in python

http://stackoverflow.com/a/14247383